app/src/main/java/com/sun/cookbook/screen/home/recyclerview/HomeRecyclerViewAdapter.kt

```diff
@@ -0,0 +1,33 @@
+package com.sun.cookbook.screen.home.recyclerview
+
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import androidx.recyclerview.widget.RecyclerView
+import com.sun.cookbook.R
+import com.sun.cookbook.data.model.RecipeRecyclerView
+
+class HomeRecyclerViewAdapter(private val onItemClicked: (RecipeRecyclerView) -> Unit) :
```

Comment:
Đặt tên thì nên đặt tên thể hiện chức năng của nó 
Adapter này phục vụ chuyển dữ liệu về Recipe thành các ItemView 
Nếu màn hình Home có nhiều RecyclerView thì đặt như nào ?
```suggestion
class RecipeAdapter(private val onItemClicked: (RecipeRecyclerView) -> Unit) :
```